mina.go

```diff
@@ -116,6 +117,18 @@ func (m *Mina) ServeHTTP(wr http.ResponseWriter, req *http.Request) {
 	req.Header.Del("If-Modified-Since")
 	req.Header.Del("If-None-Match")
 
+	if req.Header.Get(XHeaderName) == XHeaderValueIgnore {
+		wrRecorder := httptest.NewRecorder()
+		p.ServeHTTP(wrRecorder, req)
+
+		resp := wrRecorder.Result()
+		defer resp.Body.Close()
+
+		writeHeadersToWR(wr, resp, m.Headers, XHeaderValueIgnore)
```

Comment:
Let's keep the number of possible header values small, just write XHeaderValueMiss instead.